packages/sdk/src/lib/utils/env.ts

```diff
@@ -0,0 +1,6 @@
+import * as dotenv from 'dotenv'
+import * as path from 'path'
+
+export const loadEnv = () => {
+  dotenv.config({ path: path.resolve(__dirname, '../../../../../.env') })
```

Comment:
will this work properly both in the case when you're running commands at the root level of the monorepo, and when you cd into the package and run it there?